src/views/preview/preview.jsx

```diff
@@ -65,11 +65,18 @@ class Preview extends React.Component {
         // parts[0]: 'preview'
         // parts[1]: either :id or 'editor'
         // parts[2]: undefined if no :id, otherwise either 'editor' or 'fullscreen'
+
+        // Get single-comment id from url hash, using the #comment-{id} scheme from scratch2
+        const commentHashPrefix = '#comment-';
+        const singleCommentId = window.location.hash.indexOf(commentHashPrefix) !== -1 &&
+            parseInt(window.location.hash.replace(commentHashPrefix, ''), 10);
+
         this.state = {
             extensions: [],
             favoriteCount: 0,
             loveCount: 0,
             projectId: parts[1] === 'editor' ? '0' : parts[1],
+            singleCommentId: singleCommentId,
```

Comment:
Does this belong in state? Will it change? We could probably do this where we actually render the view and pass it in as a prop.

Reply:
I was planning to leave it as state because we may add a way to clear out this view and instead just view all comments.